### ISSUE TITLE
A question about fine-tune setting for qwen2-vl-7b

### ISSUE THREAD

**Issue by xzz81** [Z1zs/MMUnlearner#7]
Dear Authors,

Thank you very much for your excellent work!

I encountered some issues when running your original code to fine-tune Qwen2-VL-7B using the MLLMU fine-tuning function on my side.

Since I did not find detailed configuration settings for the fine-tuning mode in the paper, I followed the default settings in the code:
	1.	Epoch = 1
	2.	Learning rate = the default value in the code

However, after fine-tuning, my results differ from those reported in the paper. The paper states that after fine-tuning Qwen2-VL-7B on the MMUnlearner dataset with a 5% forget ratio, both QA and VQA accuracy on the Forget and Retain splits can reach around 55%. In my experiments with the settings above, the accuracy only reaches around 45%.

Could you please share the specific fine-tuning hyperparameters you used, such as batch size, number of epochs, and learning rate? If possible, would you be willing to provide the exact configuration you used for these experiments?

**Comment by Z1zs** [comment]
Hi thanks very much for your interest in our work🙏
We have received some feedback regarding the training receipt and behaviors of the vanilla model released by MLLMU-Bench since our work was released, which suggests that there maybe a bit overfitting in the official vanilla model of MLLMU-Bench. However in our work we did test the performance of official weights on evaluation script of MLLMU-Bench. Currently we highly recommend to refer for their repo (https://github.com/franciscoliu/MLLMU-Bench) for more information. According to my memory, I  also tried to adjust the training data format(I.e., and text-only and profile-level information) and hyper parameters to make sure the performance of vanilla model behaves as expected. As for the detailed fine-tuning configurations, it’s has been a long time since the work has been published, and unfortunately I have no access to the server of company in which I stored the data and models. I will try to reach out if the data is still available anyway.
